.github/renovate.json5

```
@@ -0,0 +1,42 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended",
+        ":approveMajorUpdates",
+        ":semanticCommitScopeDisabled"
+    ],
+    "ignorePresets": [":semanticPrefixFixDepsChoreOthers"],
+    "internalChecksFilter": "strict",
+    "labels": ["dependencies"],
+
+    // Wait well over three days for any new package as a precaution against malicious publishes
```

You _can not_ use comments with plain JSON. 😉

Pick one of these solutions:

- Drop the comment and use plain JSON, or
- Convert to the `.json5` format to use comments. Renovate supports `.json5`

---

it's already a json5 file 😉

---

Whoops, I was trying the config on my fork of ESLint and accidentally used the plain JSON format...

I missed that the upstream uses JSON5 already. 🙃

---

```suggestion
    // Wait seven days for any new package as a precaution against malicious publishes.
```

The comment says: well over three days, but the Renovate config will wait seven days. The comment should match the config. 😉

---

> well over three

["Well over"](https://www.merriam-webster.com/dictionary/well%20over) implies that it's intentionally more a number much larger than three. Is this not a common idiom? (I'm constantly learning about phrases I thought were but actually aren't, heh)

---

The minimum release age is set to three in one of Renovate's presets  to prevent holding a nom package that might get unpublished in the three day window. After that its user preference how long they want to wait to upgrade.

The idiom is fine, but to me it feels like there's a missing part. 🙂 But feel free to ignore me on this.

---

Backlinking for reference: https://docs.npmjs.com/policies/unpublish#packages-published-less-than-72-hours-ago

---

@JoshuaKGoldberg let's just add that link into the comment? It did read as a little strange to me, too, but makes sense in the context of this link.